docs/pages/demo/datepicker/CustomDay.example.jsx

```diff
@@ -6,9 +6,9 @@ import startOfWeek from 'date-fns/startOfWeek';
 import TextField from '@material-ui/core/TextField';
 import isWithinInterval from 'date-fns/isWithinInterval';
 import { makeStyles } from '@material-ui/core';
-import { DatePicker, Day } from '@material-ui/pickers';
 // this guy required only on the docs site to work with dynamic date library
 import { makeJSDateObject } from '../../../utils/helpers';
+import { DatePicker, PickersDay } from '@material-ui/pickers';
```

Comment:
Ohh, look like `eslint-plugin-pretty-imports` comes with a cost: harder review and harder git diff: it's moving code around.